libraries/AP_Notify/RGBLed.cpp

```diff
@@ -88,7 +102,7 @@ void RGBLed::update_colours(void)
     }
 
     // use dim light when connected through USB
-    if (hal.gpio->usb_connected()) {
+    if (hal.gpio->usb_connected() && brightness>_led_dim) {
```

Comment:
Need to fix the white space so we can read it more easily.
&& (brightness > _led_dim)